Composer/packages/client/src/pages/design/VisualEditorHeader.tsx

```diff
@@ -85,10 +86,10 @@ const useBreadcrumbs = (projectId: string, pluginConfig?: PluginConfig) => {
   const focusPath = getFocusPath(selected, focused);
   const trigger = triggerIndex != null && dialogData.triggers[triggerIndex];
 
-  let breadcrumbArray: Array<BreadcrumbItem> = [];
+  const array: Array<BreadcrumbItem> = [];
```

Comment:
`array` is an ambiguous name! in some programming language it's even a reserved word